projects/prometheus/alertmanager/Makefile

```diff
@@ -0,0 +1,79 @@
+REPO?=alertmanager
+CLONE_URL?=https://github.com/prometheus/$(REPO).git
+GIT_COMMIT?=$(shell cat GIT_COMMIT)
```

Comment:
Any reason that we are keeping tracking of commit instead of tag?

Reply:
Yes, the Dockerfile in the v0.21.0 tag (https://github.com/prometheus/alertmanager/blob/v0.21.0/Dockerfile#L13) performs chown with a `nogroup` group which does not exist on the busybox base image. They fixed it here https://github.com/prometheus/alertmanager/blob/master/Dockerfile#L13